pillar/dockerhub.sls

```diff
@@ -0,0 +1,6 @@
+docker-registries:
+    # Dockerhub
+    # https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.dockerng.html#authentication
+    hub: 
+        username: foo
```

Comment:
I think this is implemented in https://github.com/elifesciences/builder-base-formula/blob/master/elife/docker-push.sls as authentication should cover both pushing and accessing private images. That file would have to be renamed.